src/main/java/com/hotels/housekeeping/service/impl/FileSystemHousekeepingService.java

```diff
@@ -47,39 +48,65 @@ public FileSystemHousekeepingService(LegacyReplicaPathRepository legacyReplicaPa
     LOG.warn("{}.fixIncompleteRecord(LegacyReplicaPath) should be removed in future.", getClass());
   }
 
+  private FileSystem fileSystemForPath(LegacyReplicaPath cleanUpPath) {
+    LOG.info("Attempting to delete path '{}' from file system", cleanUpPath);
+    Path path = new Path(cleanUpPath.getPath());
+    FileSystem fs;
+    try {
+      fs = path.getFileSystem(conf);
```

Comment:
You could just return this and then you don't even need the "fs" variable. The scope expanding out on line 54 definitely isn't needed.

Reply:
Done